src/main/java/ch/njol/skript/sections/SecLoop.java

```diff
@@ -39,6 +40,43 @@
 import java.util.Map;
 import java.util.WeakHashMap;
 
+@Name("Loop")
+@Description({
+	"Loop sections repeat their code with multiple values.",
+	"",
+	"A loop will loop through all elements of the given expression, e.g. all players, worlds, items, etc. " +
```

Comment:
Any reason you're using `+` here as opposed to adding more items (e.g. using a comma)

Reply:
So it looks like this
![image](https://user-images.githubusercontent.com/20037329/174401596-537d899a-2a4f-4a6d-9340-b1dbac85186b.png)

I got this description from [Njol's docs](http://en.njol.ch/projects/skript/doc/loops)

Reply:
Got it! 👍